recipe/patches/0022-make-symlinks-for-python.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/pypy/tool/release/package.py b/pypy/tool/release/package.py
```

Comment:
I didn't add this file for now because we already have the symlinks in the python meta package. Once there's a new version release, we can add the symlinks into this package.

Reply:
The symlinks are not present when I use the latest build. I don't understand the interplay between this repo and the python meta package, but something is not working.